stable/prometheus-mongodb-exporter/templates/role.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: {{ template "prometheus-mongodb-exporter.fullname" . }}
+  labels:
+    app: {{ template "prometheus-mongodb-exporter.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+{{- end }}
```

Comment:
What's the point of this Role? It has no `rules` section, making it useless. I don't see the point of the two rbac files you added or `rbac.create`. I would suggest you remove them and just keep the serviceAccount logic.